package.json

```diff
@@ -41,6 +41,7 @@
     "devDependencies": {
         "@octokit/rest": "latest",
         "@types/chai": "latest",
+        "@types/fancy-log": "^2.0.0",
```

Comment:
These new dev-deps might cause a hump when bisecting, as they are needed to compile the scripts and I do that before testing.

Maybe we shouldn't compile the scripts on test, and instead do that... some other time?

Reply:
I would do it just in CI.